src/resources/views/crud/columns/color.blade.php

```diff
@@ -0,0 +1,28 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['escaped'] = $column['escaped'] ?? true;
+    $column['text'] = $column['default'] ?? '#000000';
+
+    if($column['value'] instanceof \Closure) {
+        $column['value'] = $column['value']($entry);
+    }
+
+    if(is_array($column['value'])) {
+        $column['value'] = json_encode($column['value']);
+    }
+
+    if(!empty($column['value'])) {
+        $column['text'] = $column['value'];
+    }
+@endphp
+
+<span>
```

Comment:
Let's show the color (good) but also show the color text (for copy-paste):

<img width="111" alt="CleanShot 2022-09-27 at 14 24 32@2x" src="https://user-images.githubusercontent.com/1032474/192513026-9403b3f4-c5bb-4226-8e3f-ec8ba6773a39.png">